public/locales/hu/translation.json

```diff
@@ -115,7 +115,7 @@
     "exit-to-tray": "Kilépés a Rendszer Tálcára",
     "gamemode": "GameMode használata (Feral Game Mode-nak telepítve kell lennie)",
     "language": "Válasszd ki az Alkalmazás Nyelvét",
-    "mangohud": "Engedélyezd a Mangohud-ot (Mangohud-nak telepítve kell lennie)",
+    "mangohud": "Mangohud engedélyezése (Mangohud-nak telepítve kell lennie)",
```

Comment:
In hungarian, there is a huge difference between imperative sentence and declarative sentence.
I first thought it would be an imperative sentence, as you have to enable mangohud, not as an option to enable or disable it.
Tthen I figured out I can real-time debug Heroic with yarn, so I continued translating not just watching the file, but the place of the text. And this mistake was left here :( 
sry